source/help/apps/desktop-changelog.rst

```diff
@@ -1,6 +1,73 @@
 Desktop Application Changelog
 ========================================
 
+Release v4.4.0
+----------------------------
+
+**Download Binaries:** 
+
+- **v4.4.0, release day 2020-02-16**
+
+
+**Breaking Changes** 
+XXX Can be removed if there are none XXX
+
+
+Improvements
+~~~~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Added support for PIV Card Certificate Authentication.
+- Added support for Desktop App Window/Tab UI Updates.
```

Comment:
"Added support" doesn't really make sense for UI updates. And we should try to hit on the benefits, such as drag and drop and dark theme support. Can we try rewording?